cpp/src/arrow/ipc/CMakeLists.txt

```diff
@@ -56,6 +56,7 @@ set(FBS_SRC
   ${CMAKE_SOURCE_DIR}/../format/File.fbs
   ${CMAKE_SOURCE_DIR}/../format/Schema.fbs
   ${CMAKE_SOURCE_DIR}/../format/Tensor.fbs
+  ${CMAKE_SOURCE_DIR}/../format/SparseTensor.fbs
```

Comment:
What do you think about including this definition in Tensor.fbs also?

Reply:
OK. I'll move the definition of SparseTensor in Tensor.fbs.